### ISSUE TITLE
Wine build doesn't work without mingw

### ISSUE THREAD

**Issue by cidkidnix** [Guy1524/wine#7]
- This is a known caveat, just making a issue for tracking

**Comment by Guy1524** [comment]
Yes, this is due to the NumberOfSections field in the fake dll not matching the field in memory.  A hack would be simply to increase the section count in the fake dll, but without actually adding fake sections, this may cause crashes when different code analyzes the file.

**Comment by cidkidnix** [comment]
So is there a patchset/patches that you could provide or give instructions to how to implement the change and list it as a "fix" for without mingw at your own risk
